### PR TITLE
add markdown editor on event description

### PR DESCRIPTION
**Pull request by lucianosousa**

### CODE REVIEW

**Comment by tauil** [comment]
### No visual improvement at all if I use markdown in a proposal
screenshot from 2017-04-10 12-46-53

### Headers and lists becomes black in the event description
screenshot from 2017-04-10 12-42-24

**Comment by lucianosousa** [comment]
I think we should improve the layout in this case showing the description in other place, outside an small div and a `<p>` tag

**Comment by lucianosousa** [comment]
it's already on `feature/new-ux` branch. we can think about it in this new layout

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -23,4 +23,26 @@ def render_flash_notice
...
+      fenced_code_blocks: true
+    }
+
+    extensions = {
+      autolink:           true,
+      superscript:        true,
+      disable_indented_code_blocks: true
+    }
+
+    @renderer ||= Redcarpet::Render::HTML.new(options)
```
Comment (lucasrenan) [inline app/helpers/application_helper.rb]: why renderer is a class variable?

Comment (lucianosousa) [inline app/helpers/application_helper.rb]: @lucasrenan it was the way I found to render markdown as html on view page. Feel free to suggest any improvement here. :)

**Comment by lucianosousa** [comment]
@tauil looks like this PR have sense just with the new layout. That's why I want to merge it on this branch

**Comment by lucianosousa** [comment]
@tauil ` when editing a proposal, when you try to save it raises an error 500 (some redirect error)`
didn't receive this error here, can you show me the stack trace?

**Comment by tauil** [comment]
Just for the record, no errors raising for me anymore and I fixed the markdown texts layout

**Pull request merged**